geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/ScaldingDelimitedIngestJob.scala

```diff
@@ -127,7 +128,7 @@ class ScaldingDelimitedIngestJob(args: Args) extends Job(args) with Logging {
   // Check to see if this an actual ingest job or just a test.
   if (!isTestRun) {
     new MultipleUsefulTextLineFiles(pathList: _*).using(new Resources)
-      .foreach('line) { (cres: Resources, line: String) => lineNumber += 1; ingestLine(cres.fw, line) }
+      .foreach('line) { (cres: Resources, line: String) => lineNumber += 1; ingestLine(cres.fw, line, skipHeader) }
```

Comment:
perhaps here would be the best place to do the if statement:
```
.foreach('line) { (cres: Resources, line: String) => 
   lineNumber += 1; 
   if (lineNumber != 1 || !skipHeader) {
    ingestLine(cres.fw, line) 
   }
}
```
This avoids exception handling for flow control which is much slower than if/else